ocs_ci/utility/localstorage.py

```diff
@@ -1,20 +1,21 @@
 """
 This module contains local-storage related methods
 """
+import json
 import logging
 import os
 import shutil
 import yaml
-import json
 
-from ocs_ci.utility.utils import clone_repo, run_cmd
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants, defaults
 from ocs_ci.ocs.node import get_typed_nodes
-from ocs_ci.utility.retry import retry
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs.resources import csv
 from ocs_ci.utility import utils
-from ocs_ci.ocs import constants, defaults
-from ocs_ci.framework import config
+from ocs_ci.utility.retry import retry
+from ocs_ci.utility.utils import clone_repo, run_cmd
```

Comment:
We are already importing the entire `utils` module on line 16. Can we not use `utils.clone_repo` and `utils.run_cmd` or change the `utils.<utility>` calls to be explicitly imported? We probably shouldn't be doing both.

Reply:
done